tools/dgeni/processors/merge-inherited-properties.ts

```diff
@@ -52,8 +52,18 @@ export class MergeInheritedProperties implements Processor {
       // by using an instance comparison.
       // tslint:disable-next-line:ban Need to use Object.assign to preserve the prototype.
       const newMemberDoc = Object.assign(Object.create(memberDoc), memberDoc);
+
+      // Dgeni won't add the `description` if the member doc belongs to a class
```

Comment:
I've looked more into this as it surprised me that the Dgeni TS package would process inherited classes differently. It looks like the docs miss the `description` field because we manually construct the `ClassExportDoc` for inherited classes, and these docs are not processed e.g. by the `dgeni-packages/jsdoc` processors, hence missing the various tag field and description. 

I wonder if there is a better solution to this; this is definitely better than using `content` for now, but it's rather a workaround for a more general problem that could cause issues in other places (e.g. if we test for JSDoc tags elsewhere):